first-analysis-steps/code/decay-tree-fitter/ntuple_DTF1.py

```diff
@@ -1,7 +1,7 @@
 from GaudiConf import IOHelper
 from Configurables import (
     DaVinci,
-    DecayTreeTuple,
+    DecayTreeTuple
```

Comment:
This should be kept to follow the generally preferred Python style:
```suggestion
    DecayTreeTuple,
```
The motivation is that adding a new line avoids a two-line diff. It also minimised bugs like:

```python
words = [
    "I",
    "am",
    "a"
    "list",
    "of",
    "words",
]
```

Reply:
My reason for changing this initially was that during testing this I found that `lb-ap test ...` wasn't working with this extra comma, and that removing it fixed it. However, trying it again now, it seems fine - not sure what happened, but it works!